Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -41,16 +41,19 @@ NumParticlesToAdd(const Box& overlap_box, const RealBox& overlap_realbox,
 	
 	int ref_num_ppc = num_ppc * AMREX_D_TERM(fac, *fac, *fac);
 	for (int i_part=0; i_part<ref_num_ppc;i_part++) {
-	    std::array<Real, 3> r;
-	    plasma_injector->getPositionUnitBox(r, i_part, fac);
+	    std::array<Real, 3> point;
+	    plasma_injector->getPositionUnitBox(point, i_part, fac);
+	    Real x = overlap_corner[0] + (iv[0] + point[0])*dx[0];
 #if ( AMREX_SPACEDIM == 3 )
-	    Real x = overlap_corner[0] + (iv[0] + r[0])*dx[0];
-	    Real y = overlap_corner[1] + (iv[1] + r[1])*dx[1];
-	    Real z = overlap_corner[2] + (iv[2] + r[2])*dx[2];
+	    Real y = overlap_corner[1] + (iv[1] + point[1])*dx[1];
+	    Real z = overlap_corner[2] + (iv[2] + point[2])*dx[2];
 #elif ( AMREX_SPACEDIM == 2 )
-	    Real x = overlap_corner[0] + (iv[0] + r[0])*dx[0];
 	    Real y = 0;
-	    Real z = overlap_corner[1] + (iv[1] + r[1])*dx[1];
+#ifdef WARPX_RZ
+	    Real z = overlap_corner[1] + (iv[1] + point[2])*dx[1];
```

Comment:
Could you add a comment explaining the difference between RZ and 2D here? Because of the indices (`1`, `2`), it is not entirely transparent what quantity corresponds to `r`, `z` or `theta`.